apps/total-typescript/src/pages/api/sanity/transcript/update.ts

```diff
@@ -0,0 +1,117 @@
+import {withSentry} from '@sentry/nextjs'
+import {NextApiRequest, NextApiResponse} from 'next'
+import sanityClient from '@sanity/client'
+import {json} from 'stream/consumers'
+import {uniqueId} from 'lodash'
+import {randomUUID} from 'crypto'
+
+const addTranscriptToVideoDocument = async (
```

Comment:
is this a casting words webhook receiver?

if so it should be `/api/skill/transcript/castingwords`

what is doing with the transcript is a second order concern likely by a "service"